src/app/app.ts

```diff
@@ -56,6 +56,7 @@ const creator = new SlashCreator({
 	applicationID: process.env.DISCORD_BOT_APPLICATION_ID,
 	publicKey: process.env.DISCORD_BOT_PUBLIC_KEY,
 	token: process.env.DISCORD_BOT_TOKEN,
+	disableTimeouts: true,
```

Comment:
What is this doing, exactly? Any other side effects?

Reply:
Timeout differs the reply if it takes more than 2 seconds. I have added it back exactly in src/app/activity/bounty/ActivityHandler.ts. There are no side-effects for this

Reply:
OK, so by recreating the functionality you bypassed the discord error you were getting?

Reply:
Right

Reply:
Basically, we disabled that functionality from slash-create and added it back from our end